ajax/libs/tabulator/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "tabulator",
+  "version": "2.10.0",
+  "description": "An easy to use table generation JQuery UI Plugin",
+  "filename": "tabulator.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/olifolkerd/tabulator.git"
+  },
+  "keywords": [
+    "table",
+    "grid",
+    "datagrid",
+    "tabulator",
+    "editable",
+    "cookie",
+    "jquery",
+    "jqueryui",
+    "sort",
+    "format",
+    "resizable",
+    "list",
+    "scrollable",
+    "ajax",
+    "json",
+    "widget"
+  ],
+  "author": "Oli Folkerd",
+  "license": "MIT",
+  "homepage": "https://olifolkerd.github.io/tabulator/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/olifolkerd/tabulator.git",
+    "fileMap": [
+      {
+        "basePath": "themes",
+        "files": [
+          "**/*.css"
```

Comment:
@clairetsai818 
- Have you tried to use auto-updater to fetch files with this config? It will not fetch `tabulator.js` and `tabulator.css` under root of the repo.
- It's not good to use `themes` as basePath.